components/omega/doc/design/VertCoord.md

```diff
@@ -0,0 +1,292 @@
+(omega-design-vert-coord)=
+# Vertical Coordinate
+
+## 1 Overview
+The vertical coordinate module will be responsible for computing and storing information related to the vertical mesh in Omega.
+Since Omega will be a non-Boussinesq model, the vertical independent variable will be pressure- as opposed to $z$-based.
+Similar to MPAS-Ocean's support for tilted height ($z^\star$) coordinates, Omega will support a general ($p^\star$) vertical coordinate.
+The prognostic variable in the layered continuity equation is mass thickness, which can be used to calculate the total pressure at a given vertical level.
+The vertical height of a given level is still necessary in the model to compute sea level, the geopotential, and derivatives with respect to $z$ (vertical mixing, etc).
```

Comment:
I don't think vertical mixing is a good example.  It doesn't really require z derivatives that's just what we are used to